CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [v0.9.0] - 2022-02-03
+
+v0.9.0 is technically a breaking release because of the `Debug` impl of the websocket server `Builder` struct changed. In practise it should be a non-breaking upgrade for most users.
```

Comment:
```suggestion
v0.9.0 is technically a breaking release because of the `Debug` bound of `IdProvider` trait changed in the WebSocket server. In practice it should be a non-breaking upgrade for most users.
```